packages/shared/providers.yaml

```diff
@@ -1321,6 +1321,18 @@ microsoft-tenant-specific:
     refresh_params:
         grant_type: refresh_token
     docs: https://docs.nango.dev/integrations/all/microsoft-tenant-specific
+microsoft-ads:
+    alias: microsoft-teams
+    categories:
+        - marketing
+    default_scopes:
+        - https://ads.microsoft.com/msads.manage
+        - offline_access
+    proxy: null
```

Comment:
`proxy: null` is not allowed actually. Just remove this line, then the proxy value of the aliased provider will be used